seed/data_importer/utils.py

```diff
@@ -59,234 +59,293 @@ def chunk_iterable(iterlist, chunk_size):
-        factors["District Steam"]["kLbs (thousand pounds)"] = 1194.00000000000
-        factors["District Steam"]["MLbs (million pounds)"] = 1194000.00000000000
-        factors["District Steam"]["therms"] = 100.00000000000
-        factors["District Steam"]["GJ"] = 947.81700000000
-        factors["District Steam"]["kg"] = 2.63200000000
-        factors["District Hot Water"]["kBtu"] = 1.00000000000
-        factors["District Hot Water"]["MBtu/MMBtu (million Btu)"] = 1000.00000000000
-        factors["District Hot Water"]["Therms"] = 100.00000000000
-        factors["District Hot Water"]["GJ"] = 947.81700000000
-        factors["District Chilled Water"]["kBtu"] = 1.00000000000
-        factors["District Chilled Water"]["MBtu/MMBtu (million Btu)"] = 1000.00000000000
-        factors["District Chilled Water"]["Ton Hours"] = 12.00000000000
-        factors["District Chilled Water"]["GJ"] = 947.81700000000
-        factors["Coal (anthracite)"]["kBtu"] = 1.00000000000
-        factors["Coal (anthracite)"]["MBtu/MMBtu (million Btu)"] = 1000.00000000000
-        factors["Coal (anthracite)"]["Tons"] = 25090.00000000000
-        factors["Coal (anthracite)"]["Lbs"] = 12.54500000000
-        factors["Coal (anthracite)"]["kLbs (thousand pounds)"] = 12545.00000000000
-        factors["Coal (anthracite)"]["MLbs (million pounds)"] = 12545000.00000000000
-        factors["Coal (anthracite)"]["Tonnes (metric)"] = 27658.35500000000
-        factors["Coal (anthracite)"]["GJ"] = 947.81700000000
-        factors["Coal (bituminous)"]["kBtu"] = 1.00000000000
-        factors["Coal (bituminous)"]["MBtu/MMBtu (million Btu)"] = 1000.00000000000
-        factors["Coal (bituminous)"]["Tons"] = 24930.00000000000
-        factors["Coal (bituminous)"]["Lbs"] = 12.46500000000
-        factors["Coal (bituminous)"]["kLbs (thousand pounds)"] = 12465.00000000000
-        factors["Coal (bituminous)"]["MLbs (million pounds)"] = 12465000.00000000000
-        factors["Coal (bituminous)"]["Tonnes (metric)"] = 27482.00000000000
-        factors["Coal (bituminous)"]["GJ"] = 947.81700000000
-        factors["Coke"]["kBtu"] = 1.00000000000
-        factors["Coke"]["MBtu/MMBtu (million Btu)"] = 1000.00000000000
-        factors["Coke"]["Tons"] = 24800.00000000000
-        factors["Coke"]["Lbs"] = 12.40000000000
-        factors["Coke"]["kLbs (thousand pounds)"] = 12400.00000000000
-        factors["Coke"]["MLbs (million pounds)"] = 12400000.00000000000
-        factors["Coke"]["Tonnes (metric)"] = 27339.00000000000
-        factors["Coke"]["GJ"] = 947.81700000000
-        factors["Wood"]["kBtu"] = 1.00000000000
-        factors["Wood"]["MBtu/MMBtu (million Btu)"] = 1000.00000000000
-        factors["Wood"]["Tons"] = 17480.00000000000
-        factors["Wood"]["Tonnes (metric)"] = 15857.00000000000
-        factors["Wood"]["GJ"] = 947.81700000000
-        factors["Other"]["kBtu"] = 1.00000000000
+        factors['Coal (anthracite)']['GJ'] = 947.82
```

Comment:
See documentation linked in the main PR comment/description for how these factors were created. he changes were needed as the new format had slightly different naming for many types and units. For the most part, the possible combinations stayed the same.